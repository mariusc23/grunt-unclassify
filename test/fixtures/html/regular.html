<!DOCTYPE html>
<html lang="en">
  <body>
    <div class="only"></div>
    <div class="first last"></div>
    <div class="first middle last"></div>

    <div class="exists-only"></div>
    <div class="exists-first exists-last"></div>
    <div class="exists-first exists-middle exists-last"></div>

    <div class="first exists-last"></div>
    <div class="exists-first last"></div>

    <div class="exists-first middle last"></div>
    <div class="first exists-middle last"></div>
    <div class="first middle exists-last"></div>

    <div class="exists-first exists-middle last"></div>
    <div class="first exists-middle exists-last"></div>
    <div class="exists-first middle exists-last"></div>
  </body>
</html>
