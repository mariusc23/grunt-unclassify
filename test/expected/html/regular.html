<!DOCTYPE html>
<html lang="en">
  <body>
    <div></div>
    <div></div>
    <div></div>

    <div class="exists-only"></div>
    <div class="exists-first exists-last"></div>
    <div class="exists-first exists-middle exists-last"></div>

    <div class="exists-last"></div>
    <div class="exists-first"></div>

    <div class="exists-first"></div>
    <div class="exists-middle"></div>
    <div class="exists-last"></div>

    <div class="exists-first exists-middle"></div>
    <div class="exists-middle exists-last"></div>
    <div class="exists-first exists-last"></div>
  </body>
</html>
